### ISSUE TITLE
Header is broken

### ISSUE THREAD

**Issue by brandonhaslegs** [radicle-dev/workstreams-app#139]
![](https://user-images.githubusercontent.com/4406983/176198407-64e3f47d-87ca-4d72-b1c7-5ca5ac6e950f.gif)

**Comment by efstajas** [comment]
Hey @brandonhaslegs could you clarify please? I don't really get the problem and can't reproduce what is visible in the recording

**Comment by brandonhaslegs** [comment]
Yes sorry about this vague issue. The header disappears when you scroll to the top in Safari.

**Comment by efstajas** [comment]
Fixed in #139
